modules/caddytls/connpolicy.go

```diff
@@ -260,6 +260,7 @@ func (p *ConnectionPolicy) buildStandardTLSConfig(ctx caddy.Context) error {
 	}
 
 	setDefaultTLSParams(cfg)
+	setSSLKeylog(cfg)
```

Comment:
This appears to turn on key logging regardless of whether it has been enabled or not.

Reply:
Correct. I wasn't sure how to reference the config file here.

Reply:
Could I have a bit of help on this part?

Reply:
Sure. I'll need a little bit to catch up on things first though!